### PR TITLE
Add effect for furnace form

### PR DESCRIPTION
**Pull request by CarlosFdez**

### CODE REVIEW

File: packs/feat-effects/effect-furnace-form.json [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                "key": "BaseSpeed",
+                "selector": "fly",
+                "value": 40
+            },
+            {
+                "damageType": "fire",
+                "diceNumber": 1,
+                "key": "DamageDice",
+                "predicate": [
+                    "fire"
```
Comment (stwlam) [inline packs/feat-effects/effect-furnace-form.json:57]:
```suggestion
                    "item:trait:fire"
```

File: packs/feat-effects/effect-furnace-form.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+                ],
+                "selector": "spell-damage"
+            },
+            {
+                "diceNumber": 1,
+                "key": "DamageDice",
+                "predicate": [
+                    "fire"
```
Comment (stwlam) [inline packs/feat-effects/effect-furnace-form.json:65]:
```suggestion
                    "item:trait:fire"
```

Comment (CarlosFdez) [reply packs/feat-effects/effect-furnace-form.json:65]: Seems to continue to work, even for cold blasts.

File: packs/feat-effects/effect-furnace-form.json [hunk 3]
```diff
@@ -0,0 +1,85 @@
+{
+    "_id": "Tw9MjeQHL3qFY1PO",
+    "img": "systems/pf2e/icons/spells/fiery-body.webp",
+    "name": "Effect: Furnace Form",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Furnace Form]</p>\n<p>You become living flame, giving you fire immunity, resistance 10 to precision damage, and weakness 5 to cold and to water. Any creature that touches you or damages you with an unarmed attack or non-reach melee weapon takes @Damage[3d6[fire]] damage. Your unarmed attacks deal 1d4 additional fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses). Your fire Elemental Blasts deal an additional die of damage. In fire form, you have a fly Speed of 40 feet and don't need to breathe.</p>"
```
Comment (TikaelSol) [inline packs/feat-effects/effect-furnace-form.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Furnace Form]</p>\n<p>You become living flame, giving you fire immunity, resistance 10 to precision damage, and weakness 5 to cold and to water. Your unarmed attacks deal 1d4 additional fire damage, and your fire spells deal one additional die of fire damage (of the same damage die the spell uses). Your fire Elemental Blasts deal an additional die of damage. In fire form, you have a fly Speed of 40 feet.</p>"
```
Removing stuff the effect item doesn't automate.

File: packs/feat-effects/effect-furnace-form.json [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+                    "water"
+                ],
+                "value": 5
+            },
+            {
+                "damageType": "fire",
+                "diceNumber": 1,
+                "dieSize": "d4",
+                "key": "DamageDice",
+                "label": "Fiery Body",
```
Comment (TikaelSol) [inline packs/feat-effects/effect-furnace-form.json:41]:
```suggestion
```

File: packs/feat-effects/effect-furnace-form.json [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+                "diceNumber": 1,
+                "key": "DamageDice",
+                "predicate": [
+                    "item:trait:fire"
+                ],
+                "selector": "elemental-blast-damage"
+            }
+        ],
+        "source": {
+            "value": ""
```
Comment (TikaelSol) [inline packs/feat-effects/effect-furnace-form.json:71]:
```suggestion
            "value": "Pathfinder Rage of Elements"
```

File: packs/feat-effects/effect-furnace-form.json (hunk 3 above)
Comment (CarlosFdez) [reply packs/feat-effects/effect-furnace-form.json]: The effect for fiery body has that text. Also even though it doesn't automate it, isn't it helpful information when someone mouses over the effect panel to see the description?

**Pull request merged**